packages/composerize-website/src/components/TextInput.js

```diff
@@ -3,8 +3,9 @@ import styled from 'styled-components';
 
 import CBox from './CBox';
 
-const StyledInput = styled.input`
+const StyledInput = styled.textarea`
     ${CBox}
+	height:8em;
```

Comment:
this introduces a visual change

<img width="1013" alt="Screenshot 2023-10-02 at 8 13 48 AM" src="https://github.com/magicmark/composerize/assets/1590756/64b6411c-c2cf-43e9-b693-695389b6c7f8">

I think most commands would probably be one liners, but I like the idea of making this a textarea for sure so folks can drag it down

maybe we can start this off with a `rows={1}` or 2 instead so it's not taking up quite as much room by default?

Reply:
Set 3 lines, may be could also add a second textarea to merge an existing compose with a new docker run command